packages/tools/client-debugger/client-debugger/src/messaging/Utilities.ts

```diff
@@ -16,20 +17,24 @@ import { ISourcedDebuggerMessage } from "./Messages";
  *
  * @internal
  */
-export function postMessagesToWindow<TMessage extends ISourcedDebuggerMessage>(
+export function postMessagesToWindow<TMessage extends IDebuggerMessage>(
 	loggingOptions?: MessageLoggingOptions,
 	...messages: TMessage[]
 ): void {
+	const messagesWithSource: ISourcedDebuggerMessage[] = messages.map((m) => ({
```

Comment:
Nit: Can we name the transient variable "message" or "unsourcedMessage" instead of "m" to be a bit more descriptive?